using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -525,23 +599,23 @@ ACPIdentity.getUrlVariables().then(urlVariables => console.log("AdobeExperenceSD
 {% endtab %}
 {% endtabs %}
 
-## Get identifiers
+## Get identifiers<a id="getIdentifiersTitle"></a>
 
 {% tabs %}
 {% tab title="Android" %}
-### getIdentifiers
-
-Returns all customer identifiers that were previously synced with the Adobe Experience Cloud.
+### getIdentifiers<a id="getIdentifiers-java"></a>
 
-The values are returned through the [AdobeCallback](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/identity/identity-api-reference#adobecallback).
+This `getIdentifiers()` API returns all customer identifiers that were previously synced with the Adobe Experience Cloud through the [AdobeCallback](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/identity/identity-api-reference#adobecallback).
```

Comment:
- Please reword to "This API returns all..."

- Is "Adobe Experience Cloud" referring to the v5 SDKs? If yes, this is the old name and needs to be changed to "Experience Platform Mobile SDKs".